godot-core/src/builtin/projection.rs

```diff
@@ -959,3 +959,52 @@ mod test {
         }
     }
 }
+
+impl std::fmt::Display for Projection {
+    /// Formats `Projection` to match Godot's string representation.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use godot::prelude::*;
+    /// let proj = Projection::new([
```

Comment:
```suggestion
    /// # Example
    /// ```
    /// use godot::prelude::*;
    /// let proj = Projection::new([
```